libs/pilight/protocols/433.92/kerui_d026.c

```diff
@@ -92,6 +95,7 @@ void keruiD026Init(void) {
 	options_add(&kerui_D026->options, "t", "opened", OPTION_NO_VALUE, DEVICES_STATE, JSON_STRING, NULL, NULL);
 	options_add(&kerui_D026->options, "f", "closed", OPTION_NO_VALUE, DEVICES_STATE, JSON_STRING, NULL, NULL);
 	options_add(&kerui_D026->options, "a", "tamper", OPTION_NO_VALUE, DEVICES_STATE, JSON_STRING, NULL, NULL);
+//	options_add(&kerui_D026->options, "b", "battery", OPTION_NO_VALUE, DEVICES_VALUE, JSON_STRING, NULL, NULL);
```

Comment:
The config fails because this line is commented 👍 Remove it and it works fine

Reply:
It fails when I try to start it with an existing config without the battery entry ("/devices.c #1443 ERROR: config device setting "battery" of "XY", missing ERROR: failed to read config").
So I think this change would require to manually migrate an existing config and add this entry.

Reply:
True